workflows/publishVSCode.yml

```
@@ -0,0 +1,47 @@
+name: Publish VSCode Extension
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
```

it doesn't look like we have `.nvmrc` in CVS, so let's specify a version

```suggestion
          node-version: 'latest'
```